exporter/context.go

```diff
@@ -758,7 +758,7 @@ func (ic *importContext) generateAndWriteResources(sh *os.File) {
 	for i, r := range resources {
 		ic.waitGroup.Add(1)
 		resourcesChan <- r
-		if i%50 == 0 {
+		if i%500 == 0 {
```

Comment:
I am guessing this is because there are too many items in logs so we want to minimise them?

Reply:
yes, it's just too much